### PR TITLE
Add new yaml

### PR DESCRIPTION
**Pull request by sakshigupta93**

### CODE REVIEW

File: examples/inference-deployments/mpt/mpt_30b.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+command: |
+  export PYTHONPATH=/code/llm-foundry:/code/examples:/code
+integrations:
+- integration_type: git_repo
+  git_repo: mosaicml/examples
+  git_commit: dcc4c4dff002d282a2f432ccfcab4b61dc6a53af
+- integration_type: git_repo
+  git_repo: mosaicml/llm-foundry
+  git_commit: 2c92faa5ce31888214bdb582ac7f5756d0d3dacd
+  ssh_clone: false
```
Comment (ajaysaini725) [inline examples/inference-deployments/mpt/mpt_30b.yaml:16]: Double check both this and the other commit hash before merging - make sure it completely works

File: examples/inference-deployments/mpt/mpt_30b_instruct_ft.yaml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  gpus: 1
+  instance: oci.vm.gpu.a10.1
+image: mosaicml/inference:0.1.1
+replicas: 1
+command: |
+  export PYTHONPATH=/code/llm-foundry:/code/examples:/code
+integrations:
+- integration_type: git_repo
+  git_repo: mosaicml/examples
+  git_commit: dcc4c4dff002d282a2f432ccfcab4b61dc6a53af
```
Comment (ajaysaini725) [inline examples/inference-deployments/mpt/mpt_30b_instruct_ft.yaml:12]:
```suggestion
  git_commit: dcc4c4dff002d282a2f432ccfcab4b61dc6a53af
  ssh_clone: false
```

File: examples/inference-deployments/mpt/mpt_30b_instruct_ft.yaml [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+- integration_type: git_repo
+  git_repo: mosaicml/llm-foundry
+  git_commit: 2c92faa5ce31888214bdb582ac7f5756d0d3dacd
+  ssh_clone: false
```
Comment (ajaysaini725) [inline examples/inference-deployments/mpt/mpt_30b_instruct_ft.yaml:16]: Same comment here about double checking commit hashes

**Review by ajaysaini725 — DISMISSED** [review]
Left some comments - definitely need to thoroughly test these

File: examples/inference-deployments/mpt/mpt_30b_instruct_ft.yaml [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+  gpus: 2
+  gpu_type: a100_40gb
+image: mosaicml/inference:0.1.1
+replicas: 1
+command: |
+  export PYTHONPATH=/code/llm-foundry:/code/examples:/code
+integrations:
+- integration_type: git_repo
+  git_repo: mosaicml/examples
+  git_commit: dcc4c4dff002d282a2f432ccfcab4b61dc6a53af
```
Comment (ajaysaini725) [inline examples/inference-deployments/mpt/mpt_30b_instruct_ft.yaml:12]:
```suggestion
  git_commit: 27230fa1c883c17a2a45a3762239ff7f5b0ef8ce
```

File: examples/inference-deployments/mpt/mpt_30b.yaml [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+  gpus: 2
+  gpu_type: a100_40gb
+image: mosaicml/inference:0.1.1
+replicas: 1
+command: |
+  export PYTHONPATH=/code/llm-foundry:/code/examples:/code
+integrations:
+- integration_type: git_repo
+  git_repo: mosaicml/examples
+  git_commit: dcc4c4dff002d282a2f432ccfcab4b61dc6a53af
```
Comment (ajaysaini725) [inline examples/inference-deployments/mpt/mpt_30b.yaml:12]:
```suggestion
  git_commit: 27230fa1c883c17a2a45a3762239ff7f5b0ef8ce
```

**Comment by sakshigupta93** [comment]
Verified that I can deploy mpt 7b on r7z15 with a10s using these git commits.

**Review by ajaysaini725 — APPROVED** [review]
🚀

**Pull request merged**